openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1109,6 +1109,7 @@ The following parameters are used in the JSON-encoded Credential Response body:
 * `credentials`: OPTIONAL. Contains an array of one or more issued Credentials. It MUST NOT be used if the `transaction_id` parameter is present. The elements of the array MUST be objects. This specification defines the following parameters to be used inside this object:
    * `credential`: REQUIRED. Contains one issued Credential. It MAY be a string or an object, depending on the Credential Format. See Appendix A for the Credential Format-specific encoding requirements.
 * `transaction_id`: OPTIONAL. String identifying a Deferred Issuance transaction. This parameter is contained in the response if the Credential Issuer cannot immediately issue the Credential. The value is subsequently used to obtain the respective Credential with the Deferred Credential Endpoint (see (#deferred-credential-issuance)). It MUST not be used if the `credentials` parameter is present. It MUST be invalidated after the Credential for which it was meant has been obtained by the Wallet.
+* `interval`: OPTIONAL. Contains a number that represents the minimum amount of time in seconds that the Wallet needs to wait before providing a new request to the Deferred Credential Endpoint. It MUST NOT be used if the `credentials` parameter is present. If not present, the Wallet MUST use `86400` as the default value.
```

Comment:
```suggestion
* `interval`: OPTIONAL. Contains a positive number that represents the minimum amount of time in seconds that the Wallet needs to wait before providing a new request to the Deferred Credential Endpoint. It MUST NOT be used if the `credentials` parameter is present. If not present, the Wallet MUST use `86400` as the default value.
```

Reply:
Are we comfortable with implying the default value of 1 day as the interval here, I would rather we make the interval required in the response so the implementation is always explicit about this interval.

Reply:
I would be ok making it mandatory, anybody has objections?

Reply:
interval becomes kind of redundant if we have push notifications -> I think keeping it optional is the cleaner approach to allow for that

Reply:
You would still need polling even with push, hence I think making it mandatory would be better to reduce/optimize server load.

Reply:
I think mandatory is good. Slightly simplifies the client code and slightly increases interoperability / reduces the number of variants that have to be conformance tested.

Reply:
added the comment and made it mandatory